skillNer/utils.py

```diff
@@ -43,17 +44,20 @@ def grouper(self, iterable, dist):
 
     def get_clusters(self, co_oc):
         clusters = []
-        for i, row in enumerate(co_oc):
+        i=0
+        for row in co_oc.tolil().rows:
```

Comment:
we better refactor this ``for`` loop:
1. handle the opposite case
```python
for ...:
    if len(row) == 0:
        continue
    # the rest of implementation
```
2. use enumerate co_oc.tolil().rows

Reply:
done the changes